war/src/main/webapp/css/style.css

```diff
@@ -75,68 +75,77 @@ body {
 }
 
 #side-panel {
-  z-index: 1;
```

Comment:
`z-index` was added for a reason. I can't remember why atm. Is it causing an issue of some form?